openquake/mbt/notebooks/sources_shallow_fault/mfd_double_truncated_from_slip_rate_SRC.ipynb

```diff
@@ -43,7 +43,9 @@
     "from openquake.mbt.tools.faults import get_rate_above_m_cli\n",
     "from openquake.hazardlib.geo.surface import SimpleFaultSurface\n",
     "from openquake.hazardlib.mfd import EvenlyDiscretizedMFD\n",
-    "from openquake.hazardlib.scalerel.wc1994 import WC1994"
+    "from openquake.hazardlib.scalerel.wc1994 import WC1994\n",
+    "from openquake.hazardlib.scalerel.leonard2014 import (Leonard2014_Interplate, \n",
```

Comment:
I am not sure this is necessary because of this https://github.com/GEMScienceTools/oq-mbtk/pull/110/files#diff-0505bb2590faf30962f398b42029786eR104
but I might be wrong.

Reply:
yes it's true, I will remove it